test/hotspot/jtreg/compiler/intrinsics/chacha/TestChaCha20.java

```diff
@@ -79,26 +86,32 @@ public static void main(String... args) throws Exception {
             }
 
             // Otherwise, select the tests that make sense on current platform.
-            if (containsFuzzy(cpuFeatures, "avx512")) {
+            if (containsFuzzy(cpuFeatures, "avx512", false)) {
                 System.out.println("Setting up AVX512 worker");
                 configs.add(List.of("-XX:UseAVX=3"));
             }
-            if (containsFuzzy(cpuFeatures, "avx2")) {
+            if (containsFuzzy(cpuFeatures, "avx2", false)) {
                 System.out.println("Setting up AVX2 worker");
                 configs.add(List.of("-XX:UseAVX=2"));
             }
-            if (containsFuzzy(cpuFeatures, "avx")) {
+            if (containsFuzzy(cpuFeatures, "avx", false)) {
                 System.out.println("Setting up AVX worker");
                 configs.add(List.of("-XX:UseAVX=1"));
             }
         } else if (Platform.isAArch64()) {
             // AArch64 intrinsics require the advanced simd instructions
-            if (containsFuzzy(cpuFeatures, "simd")) {
+            if (containsFuzzy(cpuFeatures, "simd", false)) {
                 System.out.println("Setting up ASIMD worker");
                 configs.add(new ArrayList());
             }
+        } else if (Platform.isRISCV64()) {
+            // Riscv64 intrinsics require the vector instructions
+            if (containsFuzzy(cpuFeatures, "v", true)) {
+                System.out.println("Setting up vector worker");
+                configs.add(new ArrayList());
```

Comment:
One more question: should we put `-XX:+UseRVV` in the list? Like `configs.add(new ArrayList(-XX:UseRVV));`.
Just not sure whether it will work when running on some older linux kernels without hwprobe auto-detection for RVV extension.

Reply:
it makes sense! Done.